test/bridge.jl

```diff
@@ -397,3 +397,57 @@ end
                                                 (MOI.VectorAffineFunction{Float64}, MOI.PositiveSemidefiniteConeTriangle, 0)))
     end
 end
+
+# We need to test this in a module at the top level because it can't be defined
+# in a testset. If it runs without error, then we're okay.
+module TestExternalBridge
+    using MathOptInterface
+
+    struct StrictlyGreaterThan <: MathOptInterface.AbstractScalarSet end
+    struct StrictlyGreaterBridge{T} <: MathOptInterface.Bridges.AbstractBridge end
```

Comment:
@blegat do bridges have to have a type parameter `T`? This cause me no end of problems. The only error you get is the unhelpful `Unionall` one.

Reply:
Yes, it has to, see https://github.com/JuliaOpt/MathOptInterface.jl/blob/b0477a680fc7ce274af60b0004fc4653cc03bee3/src/Bridges/singlebridgeoptimizer.jl#L46